File: smartrecyclerview/src/main/java/com/lapptelier/smartrecyclerview/PlaceHolderViewHolder.java
package com.lapptelier.smartrecyclerview;

import android.support.v7.widget.RecyclerView;
import android.view.View;

import butterknife.ButterKnife;

/**
 * com.lapptelier.smartrecyclerview.smart_recycler_view.PlaceHolderViewHolder
 * <p/>
 * ViewHolder of a generic placeholder
 *
 * Sub class of {@link RecyclerView.ViewHolder}
 *
 * @author L'Apptelier SARL
 * @date 14/09/2017
 */
public class PlaceHolderViewHolder extends RecyclerView.ViewHolder implements MyViewHolder<PlaceHolderCell>{

    public PlaceHolderViewHolder(View view){
        super(view);
        ButterKnife.bind(this, view);
    }

    @Override
    public void setItem(final PlaceHolderCell placeHolderCell) {

    }

}

File: smartrecyclerview/src/main/java/com/lapptelier/smartrecyclerview/PlaceHolderViewHolder.kt
package com.lapptelier.smartrecyclerview

import android.support.v7.widget.RecyclerView
import android.view.View

/**
 * com.lapptelier.smartrecyclerview.smart_recycler_view.PlaceHolderViewHolder
 *
 *
 * ViewHolder of a generic placeholder
 *
 * Sub class of [RecyclerView.ViewHolder]
 *
 * @author L'Apptelier SARL
 * @date 14/09/2017
 */
class PlaceHolderViewHolder(view: View) : RecyclerView.ViewHolder(view), SmartViewHolder<PlaceHolderCell> {

    override fun setItem(item: PlaceHolderCell, listener: ViewHolderInteractionListener) {
    }

    init {
    }


}
